features/facebookImport/src/locales/en/common.json

```diff
@@ -1,6 +1,7 @@
 {
     "title": "Importer",
     "source.your.facebook.data": "Source: Your facebook data",
+    "source.trustful": "Source: very trustful publication",
```

Comment:
I think this ia a placeholder text, right? I guess it can be replaced by "Facebook privacy policy" 
reference see here: https://wiki.polypoly.eu/pages/viewpage.action?pageId=101945320